tools/allowed-large-files

```diff
@@ -7,6 +7,7 @@ central/compliance/standards/metadata/pci_dss_3_2.go
 central/cve/converter/utils/testdata/*
 central/cve/fetcher/testdata/cve-list.json
 central/graphql/resolvers/generated.go
+central/networkgraph/entity/datastore/datastore_impl_test.go
```

Comment:
It's quite a big file with a lot of cases. Do you think there is a way to somehow make it easier to read? Maybe we should define it in yaml or some DSL to define SAC tests